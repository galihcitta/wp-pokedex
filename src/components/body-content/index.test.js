import React from "react";
import BodyContent from "./index";

let wrapper;

beforeEach(() => {
    wrapper = shallow(<BodyContent data-test="BodyContent-test-wrapper" />);
});

describe("Body Content Component", () => {
    it("renders and create snapshot <BodyContent />", () => {
        expect(toJson(wrapper)).toMatchSnapshot();
    });

    it("renders", () => {
        expect(wrapper.exists()).toBe(true);
    });
});
