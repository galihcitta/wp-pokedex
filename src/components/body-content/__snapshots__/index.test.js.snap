// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body Content Component renders and create snapshot <BodyContent /> 1`] = `
.c0 {
  display: block;
  padding-top: 60px;
  width: 100%;
}

<div
  className="c0"
  data-test="BodyContent-test-wrapper"
/>
`;
